sympy/printing/fortran.py

```diff
@@ -772,6 +772,23 @@ def _print_ArrayConstructor(self, ac):
         fmtstr = "[%s]" if self._settings["standard"] >= 2003 else '(/%s/)'
         return fmtstr % ', '.join((self._print(arg) for arg in ac.elements))
 
+    def _print_FunctionCall(self, expr):
+        return '{name}({args})'.format(
+            name=self._print(expr.name),
+            args=', '.join((self._print(arg) for arg in expr.function_args))
+        )
+
+    def _print_KeywordFunctionCall(self, expr):
```

Comment:
This looks good. I wonder if we should move this method to CodePrinter even? If so we should probably add a test in `test_pycode.py`:
```python
def test_KeywordFunctionCall():                                                                                                                    
    assert pycode(KeywordFunctionCall('zip', (x, y), dict(strict=True))) == "zip(x, y, strict=True)"
```
I'll let you decide if you want to do so in this PR or defer it for someone else / another PR.